canonical/ubuntu.com
# Remove instant provisioning for exams

## Done

- Remove instant provisioning for exams
- Update text on scheduling page to inform users that exams can be scheduled 30 mins or more into the future

## QA

- Visit https://ubuntu-com-13045.demos.haus/credentials/your-exams
  - Verify that there is no "Take now" button for redeemed exams

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-5127

## Screenshots

credentials/your-exams:

![image](https://github.com/canonical/ubuntu.com/assets/995051/dd746141-2106-4911-9994-1c2945dad4dd)

credentials/schedule:

![image](https://github.com/canonical/ubuntu.com/assets/995051/5a58c328-bd37-449c-a08f-19a6d47005de)


## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)
